Source/WebKit/Shared/Cocoa/PDFKitSoftLink.h

```diff
@@ -35,10 +35,13 @@ SOFT_LINK_FRAMEWORK_FOR_HEADER(WebKit, PDFKit)
 
 ALLOW_DEPRECATED_DECLARATIONS_BEGIN
 SOFT_LINK_CLASS_FOR_HEADER(WebKit, PDFAnnotationLink)
+SOFT_LINK_CLASS_FOR_HEADER(WebKit, PDFAnnotationChoiceWidget)
+SOFT_LINK_CLASS_FOR_HEADER(WebKit, PDFAnnotationPopup)
+SOFT_LINK_CLASS_FOR_HEADER(WebKit, PDFAnnotationText)
+SOFT_LINK_CLASS_FOR_HEADER(WebKit, PDFAnnotationButtonWidget)
```

Comment:
I claimed alphabetic ordering in the commit message and then I see this. 🙅 Will move around

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/71dc5eecfc468d07b8c180342f01b32041848d68